EmptySpaces.java

```diff
@@ -0,0 +1,26 @@
+// Java Program to remove all white spaces from a string without using replace().
+
+public class EmptySpaces {
+	public static void main(String[] args) {
+		String text = "This is a Java Program to remove all white spaces from a String.";
+		System.out.print(text);
+		char[] c = removeSpaces(text);
+		String noSpacesText = new String(c);
+		System.out.print("\n\n" + noSpacesText);
+	}
+
+	// Method to make a Array without any blank spaces characters in it.
+	public static char[] removeSpaces(String line) {
+		char[] charArray = line.toCharArray();
```

Comment:
logical error. if I put space in last place it will give "java.lang.ArrayIndexOutOfBoundsException".

Reply:
committed the changes